src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1805,80 +1790,65 @@ def get_individual_privacy_request_tasks(
 
 
 @router.post(
-    PRIVACY_REQUEST_TASK_QUEUE,
+    PRIVACY_REQUEST_REQUEUE,
     dependencies=[
         Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_CALLBACK_RESUME])
     ],
-    response_model=Dict,
+    response_model=PrivacyRequestResponse,
 )
-def queue_task_manually(
+def requeue_privacy_request(
```

Comment:
DSR 3.0 Core (the feature branch PR) has an endpoint to manually queue an individual Request Task if it was stuck. This was a little limited based on the Privacy Request Status, plus it meant a customer might have to determine multiple tasks that needed to be requeued, a little too much manual effort.

To simplify, I am replacing with another endpoint from which to restart a Privacy Request that is assumed to be stuck.  This should only be used for now if you believe the Privacy Request to be in a bad state, for example, it is stuck in In Processing or Approved.  You should be careful to just use this if you think it is stuck, as we generally only want the Privacy Request in the queue once which is why we are careful about all this Privacy Request Status checking. However, it's possible the Privacy Request Status is incorrect with unexpected failure states.